packages/malloy-db-bigquery/src/bigquery_connection.ts

```diff
@@ -74,12 +74,12 @@ interface CredentialBody {
 }
 
 interface BigQueryConnectionConfiguration {
-  defaultProject?: string;
+  projectId?: string;
```

Comment:
It might be good to include doc strings like `/** This ID is used for table path normalization */` here, so that consumers of the connection will get this info in their IDE.

Reply:
👍